plugins/DevicesDetection/templates/detection.twig

```diff
@@ -6,6 +6,32 @@
 
     <script type="text/javascript">
 
+        if (!navigator.userAgentData || typeof navigator.userAgentData.getHighEntropyValues !== 'function') {
+            $('#noclienthints').show();
+            $('[name=clienthints],#usech').hide();
```

Comment:
The 'Your Browser does not support client hints.' message doesn't show for me on Firefox, I think the span visibility is being toggled before page rendering is complete. Wrapping the whole `if..else` block with `$(document).ready(functino() {..});` gets the message to show.